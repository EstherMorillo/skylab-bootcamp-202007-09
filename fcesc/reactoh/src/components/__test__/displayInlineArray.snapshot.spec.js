import React from 'react';
import renderer from 'react-test-renderer';
import DisplayInlineArray from './../DisplayInlineArray';
import HERO_LIST from './../../mockdata/superHeroData';

const tree = renderer.create(<DisplayInlineArray aliases={HERO_LIST[0].biography.aliases}/>).toJSON();

describe('DisplayInlineArray snapshot', () => {

    test('should match', () => {
        expect(tree).toMatchSnapshot();
    })
})
