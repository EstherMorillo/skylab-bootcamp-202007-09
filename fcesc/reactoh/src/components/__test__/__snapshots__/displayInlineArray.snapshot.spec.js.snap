// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayInlineArray snapshot should match 1`] = `
<ul
  className="inline__array"
>
  <li
    className="inline__array"
  >
    Rick Jones
  </li>
</ul>
`;
